package.json

```diff
@@ -1,18 +1,16 @@
 {
   "name": "@okta/samples-js-react",
   "version": "1.4.0",
-  "description": "",
-  "main": "index.js",
+  "description": "React Sample Applications for Okta",
   "scripts": {
-    "test": "npm run test:e2e",
-    "test:e2e": "webdriver-manager update && protractor test/e2e/conf.js",
-    "mock-okta": "node tools/mock-okta",
-    "mock-okta:debug": "DEBUG=mock-okta npm run mock-okta",
-    "wrap": "node tools/wrap-dependencies",
-    "lint:report": "eslint -f checkstyle -o build2/reports/lint/eslint-checkstyle-result.xml --ignore-path .gitignore .",
-    "lint": "eslint --ignore-path .gitignore .",
-    "build": "webpack --config webpack.config.js",
-    "start": "node tools/dev-server"
+    "test": "eslint -c .eslintrc.json --ext .js --ext .jsx okta-hosted-login/src/ custom-login/src/ && npm run test:e2e",
+    "okta-hosted-login-server": "npm start --prefix okta-hosted-login/",
+    "test:okta-hosted-login": "protractor okta-oidc-tck/e2e-tests/okta-hosted-login/conf.js",
+    "custom-login-server": "npm start --prefix custom-login/",
+    "test:custom-login": "protractor okta-oidc-tck/e2e-tests/custom-login/conf.js",
+    "test:e2e": "npm run get-oidc-tck && npm run test:okta-hosted-login && npm run test:custom-login",
+    "get-oidc-tck": "git clone https://github.com/okta/okta-oidc-tck.git && git --git-dir=okta-oidc-tck/.git --work-tree=okta-oidc-tck checkout e2eTests",
```

Comment:
This will fail with `fatal: destination path 'okta-oidc-tck' already exists and is not an empty directory` if the repo has already been cloned

Reply:
@robertjd - Does it make sense to delete the existing directory before we clone? I kept it like this as it would never fail on travis since each travis run is in a fresh environment.

Reply:
I've moved this to a new PR which is rebased off master. Please take a look at this - https://github.com/okta/samples-js-react/pull/15